examples/core/firebase-vite/src/theme.ts

```diff
@@ -0,0 +1,13 @@
+'use client';
+import { createTheme } from '@mui/material/styles';
+import getMPTheme from './theme/getMPTheme';
+
+const lightTheme = createTheme(getMPTheme('light'));
+const darkTheme = createTheme(getMPTheme('dark'));
+
+const theme = {
```

Comment:
Can't we use a CSS variables theme instead? Also I can't find this `getMPTheme`, is this outdated?

Reply:
Yes, missed deleting this file (isn't used anywhere anymore)